packages/ai-code-completion/src/browser/ai-code-completion-preference.ts

```diff
@@ -39,6 +40,14 @@ export const AICodeCompletionPreferencesSchema: PreferenceSchema = {
                 type: 'string'
             },
             default: []
+        },
+        [PREF_AI_INLINE_COMPLETION_MAX_CONTEXT_LINES]: {
+            title: 'Maximum Context Lines',
+            type: 'number',
+            description: 'The maximum number of lines shared between the prefix and suffix for AI code completion.\
+            Set this to 0 to use the full file as context without any line limit.',
```

Comment:
This makes me a bit sad, because now there is no way to actually just have a single line, i.e., if I set 1 I get the cursor line + one above. I think it may make more sense to make `-1` the "whole document" setting and `0` the "no prefix/suffix" setting, what do you think?

Reply:
Greta suggestion, applied! thank you!